### PR TITLE
Add support for Windows Server domain user

### PR DESCRIPTION
**Pull request by mossbeard**
Add support for Windows Server domain users over windows policies.
A user need interactive rights.

### CODE REVIEW

File: src/QlikConnectorPSExecute/UserInteractive.cs [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+                    {
+                        LsaClose(policyHandle);
+                    }
+                }
+                FreeSid(sid);
+            }
+            return rights;
+        }
+
+        public void SetRight(string accountName, string privilegeName, bool remove)
```
Comment (konne) [inline src/QlikConnectorPSExecute/UserInteractive.cs]: don't repeat yourself, you can make one private Method out of Get & Set Right and make only an if around the get set lines. Call that private from the public Get / Set Method

Comment (konne) [inline src/QlikConnectorPSExecute/UserInteractive.cs]: @mossbeard please fix that, before accepting that PR

File: src/QlikConnectorPSExecute/UserInteractive.cs [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+        #endregion
+
+        #region Static Methods
+        private static bool IsLocalWinUser()
+        {
+            string strMachineName = System.Environment.MachineName;
+            return WindowsIdentity.GetCurrent().Name.ToUpper().Contains(strMachineName.ToUpper());
+        }
+
+        private static IList<string> GetRights()
```
Comment (konne) [inline src/QlikConnectorPSExecute/UserInteractive.cs]: @mossbeard 
why not a private static LocalSecurityControler, why generating in each functiona new controller

Comment (mossbeard) [inline src/QlikConnectorPSExecute/UserInteractive.cs]: It's a good idea, i still do it.

**Comment by mossbeard** [comment]
Please, check again

**Comment by konne** [comment]
why close, I just reopend that PR

**Comment by mossbeard** [comment]
Sorry, wrong button.

**Pull request merged**